composer.json

```diff
@@ -23,6 +23,7 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2 || ^0.3",
         "kriswallsmith/buzz": "^0.15",
         "psr/log": "^1.0",
+        "sensio/generator-bundle": "^3.1",
```

Comment:
Why is this required?

Reply:
Why is this required? If it is not used in the code we should remove it, it is causing some troubles, because it is not compatible with sf 4.0

@vyshkant

Reply:
@jordisala1991 As far as I remember, `sensio/generator-bundle` was required to use `Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper` and `Sensio\Bundle\GeneratorBundle\Command\Helper\DialogHelper` (see https://github.com/sonata-project/SonataMediaBundle/pull/1259#issuecomment-312598646).

But then we SUDDENLY found out that these classes are not used (see https://github.com/sonata-project/SonataMediaBundle/pull/1259#discussion_r144399174).

So now, I think, we can remove this dependency.